apps/mobile-wallet/src/screens/TransactionModal.tsx

```diff
@@ -74,40 +78,50 @@ const TransactionModal = ({ tx, ...props }: TransactionModalProps) => {
       </ScreenSectionStyled>
 
       <BoxSurface type="highlight">
-        <Row title={t('Amount')} transparent>
-          <AmountsContainer>
-            {tokensWithSymbol.map(({ id, amount, decimals, symbol }) => (
-              <AmountStyled
-                key={id}
-                value={amount}
-                decimals={decimals}
-                suffix={symbol}
-                isUnknownToken={!symbol}
-                highlight={!isMoved}
-                showPlusMinus={!isMoved}
-                fullPrecision
-                bold
-              />
-            ))}
-          </AmountsContainer>
-        </Row>
-        <Row title={t('Timestamp')} transparent>
-          <AppTextStyled semiBold>
+        {isMoved && (
+          <Row title={t('Moved')} transparent isVertical>
+            <AmountsContainer>
+              {tokensWithSymbol.map(({ id, amount }) => (
+                <AssetAmountWithLogo key={id} assetId={id} amount={amount} logoSize={18} />
+              ))}
+            </AmountsContainer>
+          </Row>
+        )}
+        {!isMoved && groupedIOAmounts.in && (
+          <Row title={t('Received')} transparent isVertical titleColor={theme.global.receive}>
+            <AmountsContainer>
+              {groupedIOAmounts.in.map(({ id, amount }) => (
+                <AssetAmountWithLogo key={id} assetId={id} amount={amount} logoSize={18} />
+              ))}
+            </AmountsContainer>
+          </Row>
+        )}
+        {!isMoved && groupedIOAmounts.out && (
+          <Row title={t('Sent')} transparent isVertical titleColor={theme.global.send}>
+            <AmountsContainer>
+              {groupedIOAmounts.out.map(({ id, amount }) => (
+                <AssetAmountWithLogo key={id} assetId={id} amount={amount} logoSize={18} />
+              ))}
+            </AmountsContainer>
+          </Row>
+        )}
```

Comment:
I would swap the order of these 2: First "Sent" and then "Received". When using dApps, it makes more sense: You send something to receive something.

Besides, if a tx includes negative amounts it means that the user has initiated the transaction. Thus, I believe, the approved amount of the user (aka: "Sent") should be displayed first, and then what they got in return for what they sent.

| I provide liquidity with these 3 tokens and I receive 0.56 LP tokens in return | I send 0.11 ALPH to swap it for 0.10 EX |
| ----- | ----- |
| <img src="https://github.com/user-attachments/assets/d41aa399-ac23-437d-8f1e-55b862256b3e" width="250"/> | <img src="https://github.com/user-attachments/assets/33d0f2b4-4a19-4e04-a972-1b1df3de7c98" width="250"/> |